proto/topo.proto

```diff
@@ -63,6 +64,7 @@ message Node {
     CISCO_XRD = 12;
     CISCO_E8000 = 13;
     LEMMING = 14;
+    CDNOS = 15;
```

Comment:
remove, this field is deprecated and no longer used, actually a good reminder for me to just delete this entirely

Reply:
removed